source/databases-collections.txt

```diff
@@ -0,0 +1,275 @@
+.. _c-databases-collections:
+
+=========================
+Databases and Collections
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: table, row, organize, storage
+
+Overview
+--------
+
+In this guide, you can learn how to interact with MongoDB databases and
+collections with the {+driver-short+}.
+
+MongoDB organizes data into a hierarchy of the following levels:
+
+- **Databases**: The top level of data organization in a MongoDB instance.
+- **Collections**: MongoDB stores documents in collections. They are analogous to tables in relational databases.
+- **Documents**: Contain literal data such as string, numbers, dates, and other embedded documents.
+
+For more information about document field types and structure, see the
+:manual:`Documents </core/document/>` guide in the {+mdb-server+} manual.
+
+Access a Database
+-----------------
+
+Access a database by using the ``mongoc_client_get_database()`` function and passing in
+a ``mongo_client_t`` instance and the name of the database as parameters.
+
+The following example accesses a database named ``"test_database"``:
+
+.. code-block:: c
```

Comment:
S: could you move the code to a separate code file and use literalincludes (for AI ingestion / code testing)?